internal/file/watch/watch.go

```diff
@@ -69,9 +71,12 @@ func (w *watch) init() (*watch, error) {
 		return nil, err
 	}
 	w.mu.RLock()
-	dirs := w.dirs
+	dirs := make([]string, 0, len(w.dirs))
+	for d := range w.dirs {
+		dirs = append(dirs, d)
+	}
```

Comment:
let me tell about why, please?

Reply:
sure.
map type is reference type in golang.
so Even if we take the lock w.dirs, it changes when the referrer changes.
When we copy the value of w.dirs, the instantaneous state is copied, so even if the referrer changes, the copy is not affected.